.gitmodules

```diff
@@ -22,6 +22,9 @@
 [submodule "TrigScint"]
 	path = TrigScint
 	url = https://github.com/LDMX-Software/TrigScint
+[submodule "Trigger"]
+	path = Trigger
+	url = https://github.com/LDMX-Software/Trigger.git
```

Comment:
Since this commit was made prior to `Tracking`, this will cause a rebase CONFLICT and will need to be resolved manually. (Easy to do, just keep both).